op-devstack/sysgo/cluster.go

```diff
@@ -3,12 +3,15 @@ package sysgo
 import (
 	"github.com/ethereum-optimism/optimism/op-devstack/shim"
 	"github.com/ethereum-optimism/optimism/op-devstack/stack"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
 	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/backend/depset"
 )
 
 type Cluster struct {
-	id     stack.ClusterID
-	cfgset depset.FullConfigSetMerged
+	id            stack.ClusterID
+	cfgset        depset.FullConfigSetMerged
+	rollupConfigs map[eth.ChainID]*rollup.Config
```

Comment:
I think `rollupConfigs` here is redundant. We have the rollupConfigs within the `L2Network`.